util/health/testdata/pod-pending.yaml

```diff
@@ -64,10 +64,6 @@ status:
     name: main
     ready: false
     restartCount: 0
-    state:
-      waiting:
-        message: Back-off pulling image "doesnt-exist"
-        reason: ImagePullBackOff
```

Comment:
Rather than remove this, can we replace it with a valid Pending condition where it's waiting but there's no error (e.g. currently waiting to be schedulable)

Reply:
You can use:
```yaml
        waiting:
          reason: PodInitializing
```

Reply:
I think this is the test case you mention above. I'm going to revert this change and update the fn to match.